### PR TITLE
Feature/working guide

### PR DESCRIPTION
**Pull request by tgalery**
Adding tips on the repo's readme on some practical tips regarding model updates.
- [x] @dav009 
- [x] roland

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -253,4 +255,31 @@ spotlightModel.getStatsForSurfaceForm("evrimleri")
...
+```
+
+## Practical tips for updating a model
+
+Given that the models are quite big (2 GB compressed), downloading, modifying and uploading them would be very
+ time consuming from your local machine. Plus, the operations require a lot of ram, so you better boot a dev instance, and do the changes from there.
+ Here's a list of steps.
+
+1. Create the new instance:
+`knife ec2 server create -r "role[spotlight]" -I ami-6d3f9704 -G default -x ubuntu --node-name "dev-spotlight" --environment "development" -f m3.xlarge --availability-zone us-east-1d`
```
Comment (dav009) [inline README.md]: lately you have to add `--secret-file ~/.chef/encrypted_data_bag_secret`

File: README.md [hunk 2]
```diff
@@ -253,4 +255,31 @@ spotlightModel.getStatsForSurfaceForm("evrimleri")
...
+
+2. ssh into the new instance and install dependencies:
+`sudo apt-get install openjdk-6-jdk maven scala unzip`
```
Comment (dav009) [inline README.md:270]: I wrote that gist before we roll to `java-8` which seems to have problems compiling (not running)  spotlight 0.6. maybe adding a head ups about modifying the `JAVA_HOME` ?

`export JAVA_HOME=/path/to/beloved/working/java`

**Comment by dav009** [comment]
:+1:

File: README.md [hunk 3]
```diff
@@ -253,4 +255,31 @@ spotlightModel.getStatsForSurfaceForm("evrimleri")
...
+```
+
+## Practical tips for updating a model
+
+Given that the models are quite big (2 GB compressed), downloading, modifying and uploading them would be very
+ time consuming from your local machine. Plus, the operations require a lot of ram, so you better boot a dev instance, and do the changes from there.
+ Here's a list of steps.
+
+1. Create the new instance:
+`knife ec2 server create -r "role[spotlight]" -I ami-6d3f9704 -G default -x ubuntu --node-name "dev-spotlight" --environment "development" -f m3.xlarge --availability-zone us-east-1d --secret-file path/to/secret/file`
```
Comment (dav009) [inline README.md]: Ive always done it with an m2.xlarge.

prd-spotlight is as well an m2.xlarge

Comment (tgalery) [inline README.md]: sheeeit, gonna change it.

**Pull request merged**